tests/system/action/motion/test_set_number_mixin.py

```diff
@@ -1082,8 +1082,8 @@ def test_complex_example_per_category_2_4(self) -> None:
         )
         self.assert_status_code(response, 200)
         motion4 = self.get_model("motion/4")
-        assert motion4.get("number") == "A 002"
-        assert motion4.get("number_value") == 2
+        assert motion4.get("number") == "A 003"
+        assert motion4.get("number_value") == 3
```

Comment:
Why did this change?

Reply:
Because the category_id handling was changed. So the category was used more times.
In the newest implementation, this is reverted again.